nautobot/dcim/navigation.py

```diff
@@ -329,6 +329,22 @@
                             ),
                         ),
                     ),
+                    NavMenuItem(
+                        link="dcim:interfaceredundancygroup_list",
+                        name="Interface Redundancy Groups",
+                        weight=600,
+                        permissions=[
+                            "dcim.view_interfaceredundancygroup",
+                        ],
+                        buttons=(
```

Comment:
Should probably add a `NavMenuImportButton` here as well. Since `NautobotUIViewSet` has a bulk_add view. The only thing missing is probably an `InterfaceRedundnacyGroupCSVForm`.

Reply:
CSV Import/Export now working (thanks for the help).

TODO: Need to discuss the import/export functionality for interfaces/associations (there are no views for the Association table, other than create/edit/delete)